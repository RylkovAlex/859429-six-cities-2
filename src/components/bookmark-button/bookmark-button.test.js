import React from 'react';
import renderer from 'react-test-renderer';
import BookmarkButton from './bookmark-button';

it(`BookmarkButton correctly renders after relaunch`, () => {
  const props = {
    buttonClasses: `css classes`,
    isFetching: false,
    handleBookmarkClick: jest.fn(),
  };

  const tree = renderer.create(<BookmarkButton {...props} />).toJSON();

  expect(tree).toMatchSnapshot();
});
