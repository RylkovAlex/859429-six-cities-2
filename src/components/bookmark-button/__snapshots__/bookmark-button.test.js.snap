// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookmarkButton correctly renders after relaunch 1`] = `
<button
  className="css classes"
  disabled={false}
  onClick={[MockFunction]}
  type="button"
>
  <svg
    className="place-card__bookmark-icon"
    height="19"
    width="18"
  >
    <use
      xlinkHref="#icon-bookmark"
    />
  </svg>
  <span
    className="visually-hidden"
  >
    To bookmarks
  </span>
</button>
`;
